File: Golang/598. Range Addition II.go
func maxCount(m int, n int, ops [][]int) int {
    if len(ops) == 0 {
        return m * n
    } else {
        for _, op := range ops {
            m = min(m, op[0])
            n = min(n, op[1])
        }
        return m * n
    }
}

func min(a, b int) int {
    if a > b {
        return b
    } else {
        return a
    }
}

File: Python3/598. Range Addition II.py
class Solution:
    def maxCount(self, m, n, ops):
        """
        :type m: int
        :type n: int
        :type ops: List[List[int]]
        :rtype: int
        """
        if not ops:
            return m * n
        op1 = (x[0] for x in ops)
        op2 = (x[1] for x in ops)
        return min(op1) * min(op2)
